packs/trident-operator-24.10.0-addon/pack.json

```diff
@@ -0,0 +1,19 @@
+{
+  "addonType":"system app",
+  "cloudTypes": [
+    "vsphere",
+    "maas",
+    "edge-native"
+  ],
+  "annotations": {
+    "docsURL": "https://github.com/NetApp/trident",
+    "description": "NetApp Trident"
+  },
+  "displayName": "Trident",
+  "charts": [
+      "charts/trident-operator-24.10.0.tgz"
+  ],
+  "layer": "addon",
+  "name": "trident",
```

Comment:
@mollusk82  - There are other versions of. trident pack available here: https://github.com/spectrocloud/pack-central/tree/main/packs/trident-operator-23.07.1 and https://github.com/spectrocloud/pack-central/tree/main/packs/trident-operator-v23.01.0
Is this another version of same pack? If so, can we have the same name (csi-trident) so that it does not list as a different pack? Ref: https://github.com/spectrocloud/pack-central/blob/main/packs/trident-operator-v23.01.0/pack.json
The older packs have layer set to 'csi'

Reply:
Hi @vishwanaths ! This pack (trident operator) is available as [CSI](https://github.com/spectrocloud/pack-central/tree/main/packs/trident-operator-23.07.1) and as [addon](https://github.com/spectrocloud/pack-central/tree/main/packs/trident-operator-23.07.1) - so users can use it in infra layer or in addon layer of their cluster profiles. This PR is to create a new version of the trident operator pack to use in addon layer.